### PR TITLE
Set encoding when saving csv

### PR DESCRIPTION
**Pull request by mski-iksm**
I've added CsvFileProcessor._encoding to allow specifying encodings when dumping in csv.

### CODE REVIEW

File: docs/task_on_kart.rst [hunk 1]
```diff
@@ -250,3 +250,25 @@ TaskOnKart.should_dump_supplementary_log_files
...
+By default, csv file is dumped with `utf-8` encoding.
+
+If you want to dump csv file with other encoding, you can use `encoding` parameter as follows:
+
+.. code:: python
+
+    def output(self):
+        return self.make_target('file_name.csv', encoding='cp932')
+        # This will dump csv as 'cp932' which is used in Windows.
+
```
Comment (hirosassa) [inline docs/task_on_kart.rst:274]: Good documentation!

**Review by hirosassa — APPROVED** [review]
LGTM

**Comment by Hi-king** [comment]
@mski-iksm Could you consider adding some light load/save test cases?

**Review by kitagry — APPROVED** [review]

File: docs/task_on_kart.rst [hunk 2]
```diff
@@ -250,3 +250,25 @@ TaskOnKart.should_dump_supplementary_log_files
 Whether to dump supplementary files (task_log, random_seed, task_params, processing_time, module_versions) or not. Default is True.
 
 Note that when set to False, task_info functions (e.g. gokart.tree.task_info.make_task_info_as_tree_str()) cannot be used.
+
+
+Dump csv with encoding
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can dump csv file by impementing `Task.output()` method as follows:
```
Comment (ukky17) [inline docs/task_on_kart.rst:258]:
```suggestion
You can dump csv file by implementing `Task.output()` method as follows: 
```

File: docs/task_on_kart.rst [hunk 3]
```diff
@@ -250,3 +250,25 @@ TaskOnKart.should_dump_supplementary_log_files
...
+You can dump csv file by impementing `Task.output()` method as follows: 
+
+.. code:: python
+
+    def output(self):
+        return self.make_target('file_name.csv')
+
+By default, csv file is dumped with `utf-8` encoding.
+
+If you want to dump csv file with other encoding, you can use `encoding` parameter as follows:
```
Comment (ukky17) [inline docs/task_on_kart.rst:267]:
```suggestion
If you want to dump csv file with another encoding, you can use `encoding` parameter as follows:
```

**Comment by ukky17** [comment]
LGTM

**Review by ukky17 — APPROVED** [review]

File: docs/task_on_kart.rst (hunk 3 above)
Comment (mski-iksm) [reply docs/task_on_kart.rst]: I've changed to `other encodings`. Thank you!

**Comment by hirosassa** [comment]
@mski-iksm just a friendly reminder. What's this PR status?

**Comment by mski-iksm** [comment]
> @mski-iksm just a friendly reminder. What's this PR status?

@hirosassa I'm still working on adding tests. Please wait for few more days.

**Comment by mski-iksm** [comment]
@Hi-king I've added small tests. Please re-review. Thank you.

**Comment by Hi-king** [comment]
@mski-iksm THX!LGTM!

**Pull request merged**